packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue

```diff
@@ -37,9 +51,13 @@ const contextLanguage = ref<string>('python3');
 const contextLanguageOptions = ref<string[]>(['python3']);
 
 const submitQuery = () => {
+	kernelStatus.value = KernelState.busy;
 	const message = props.kernelManager.sendMessage('llm_request', {
 		request: queryString.value
 	});
+	message.register('status', (data) => {
+		kernelStatus.value = data.content.execution_state;
+	});
```

Comment:
Why is it that we are checking on status-change, rather than on completion of the message-request?

Reply:
Discussed offline - there is a little big of conceptual mismatch, but it works for now.

Reply:
Post quick discussion - I am following the status. I could be making my own manual status instead. Both options work